src/main/java/basestation/bot/commands/CommandCenter.java

```diff
@@ -8,8 +8,45 @@
  * If there are any important commands we believe all robots should have then place them here
  */
 public interface CommandCenter {
+	/**
+     * Starts logging of data
+     */
+    void startLogging();
+
+    /**
+     * Returns whether or not data is currently being logged.
+     * @return True if the command center is currently logging data
+     */
+    boolean isLogging();
+
+    /**
+     * Stores key value pair. 
+     * @param name A key to identify the type of command
+     * @param value The value of the command
+     */
+    void setData(String name, String value);
 
     /**
+     * Stores the value for each wheel simultaneously.
```

Comment:
similar issue to above- we aren't just storing the command, we're sending it to the bot.